tests/dialects/test_func.py

```diff
@@ -33,7 +33,7 @@ def test_func():
             a := Constant.from_int_and_width(1, i32),
             b := Constant.from_int_and_width(2, i32),
             Addi.get(a, b)
-        ]))   # yapf: disable
+        ]))  # yapf: disable
```

Comment:
Why disable yapf?

Reply:
Because of this:

![image](https://user-images.githubusercontent.com/14147713/220969215-3a39b359-efa9-42dd-a6c8-03b6c11da0eb.png)


It's unreadable.

Reply:
You can sometimes add `# ` at the end of lines to force yapf to format differently. (I would try one before the upper `b` here.)

Reply:
Ah a comma at the end of `Addi.get(a, b)` will enforce the one line per list element